app/services/delivery_partners/participants_filter.rb

```diff
@@ -61,7 +63,13 @@ def filter_role(scoped, role)
       end
     end
 
+    def filter_out_cohorts(scoped)
+      scoped.joins(:cohort).where("start_year <= ?", LATEST_COHORT_TO_RETURN)
+    end
+
     def filter_academic_year(scoped, academic_year)
```

Comment:
I wonder if we could always filter by academic year even if filter not passed, default being filtering by all years or the one passed in, rather than adding a new `filter_out_cohorts` method?